sdk/logical/system_view.go

```diff
@@ -56,7 +56,9 @@ type SystemView interface {
 	// name. Returns a PluginRunner or an error if a plugin can not be found.
 	LookupPlugin(context.Context, string, consts.PluginType) (*pluginutil.PluginRunner, error)
 
-	NewPluginClient(ctx context.Context, pluginRunner *pluginutil.PluginRunner, config pluginutil.PluginClientConfig) (pluginutil.Multiplexer, error)
+	// NewPluginClient returns a client for managing the lifecycle of plugin
+	// processes
+	NewPluginClient(ctx context.Context, config pluginutil.PluginClientConfig) (pluginutil.PluginInstance, error)
```

Comment:
This makes me think whether we should be embedding `pluginutil.RunnerUtil` instead of duplicating the method signatures...

Reply:
Would it be more appropriate to name this `NewPluginClientInstance` now that the struct is renamed?